src/commands/remoteDebug/checkForRemoteDebugSupport.ts

```diff
@@ -0,0 +1,19 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { WebSiteManagementModels } from 'azure-arm-website';
+
+export function checkForRemoteDebugSupport(siteConfig: WebSiteManagementModels.SiteConfig): void {
+    // Read siteConfig.linuxFxVersion to determine debugging support
+    //   If the Function App is running on Windows, it will be empty
+    //   If the Function App is running on Linux consumption, it will be empty
+    //   If the Function App is running on a Linux App Service plan, it will contain Docker registry information, e.g. "DOCKER|repo.azurecr.io/image:tag"
+    //      The blessed Node.js Docker image will be something like "DOCKER|mcr.microsoft.com/azure-functions/node:2.0-node8-appservice"
+    if (siteConfig.linuxFxVersion && siteConfig.linuxFxVersion.startsWith('DOCKER|mcr.microsoft.com/azure-functions/node')) {
+        return;
+    }
+
+    throw new Error('Azure Remote Debugging is currently only supported for Node.js Function Apps running on Linux App Service plans.');
```

Comment:
Based on the PR description - this only works on dedicated plans? I feel like that should be highlighted in the messaging since consumption is so much more popular

Reply:
I looked into the conditional messaging approach but I wasn't able to find a good solution.

I went ahead and changed the messaging to be very clear, for this message and also for the description of the feature flag. Let me know what you think.